### DISCUSSION TITLE
unable to reverse proxy

### DISCUSSION THREAD

**Discussion by gilrim** [comfyanonymous/ComfyUI#3020] [General]
I'm running comfyui in a container on a vm, and when I try to put nginx in front I'm redirected to the internal ip on 192.168.20.139 rather than get the webpages proxied. From this https://github.com/comfyanonymous/ComfyUI/discussions/2786 discussion it seems to be possible, but even when just replicating the exact config in my installation I just get told to go to < location: http://192.168.20.139:1111/login . Are there some change I need to make to the caddy config in the container to allow a reverse proxy?

**Comment by beuman** [comment]
Hey did you found a solution ?
